src/main/kotlin/com/exactpro/th2/check1/rule/nomessage/NoMessageCheckTask.kt

```diff
@@ -115,5 +115,7 @@ class NoMessageCheckTask(
             resultEvent.status(Event.Status.FAILED)
                 .name("Check failed: $extraMessagesCounter extra messages were found.")
         }
+
+        resultEvent.description("Task has been completed because: {$taskState}")
```

Comment:
Maybe we should fail the check rule if the taskState == TIMEOUT or STREAM_COMPLETED because in these cases we cannot be sure if there really was not any message during the timeout the user wanted.